Makefile

```diff
@@ -0,0 +1,6 @@
+test:
+	tox
+clean:
```

Comment:
I didn't know how to use the find command like this. I will do that next time. eh!